homeassistant/const.py

```diff
@@ -1053,7 +1055,16 @@ class UnitOfVolumeFlowRate(StrEnum):
     "2025.1",
 )
 """Deprecated: please use UnitOfVolumeFlowRate.CUBIC_FEET_PER_MINUTE"""
-
+_DEPRECATED_VOLUME_FLOW_RATE_LITERS_PER_MINUTE: Final = DeprecatedConstantEnum(
+    UnitOfVolumeFlowRate.LITERS_PER_MINUTE,
+    "2025.1",
+)
+"""Deprecated: please use UnitOfVolumeFlowRate.LITERS_PER_MINUTE"""
+_DEPRECATED_VOLUME_FLOW_RATE_GALLONS_PER_MINUTE: Final = DeprecatedConstantEnum(
+    UnitOfVolumeFlowRate.GALLONS_PER_MINUTE,
+    "2025.1",
+)
+"""Deprecated: please use UnitOfVolumeFlowRate.GALLONS_PER_MINUTE"""
```

Comment:
We don't need this for new enums

Reply:
We do. We hav tests that check that these exists.

Reply:
Then there's something wrong with that test IMHO.

Reply:
Yup. Looks like a good option. Fixed.